core/templates/pages/skill-editor-page/editor-tab/skill-misconceptions-editor/skill-misconceptions-editor.directive.ts

```diff
@@ -42,13 +44,13 @@ angular.module('oppia').directive('skillMisconceptionsEditor', [
         'skill-misconceptions-editor.directive.html'),
       controller: [
         '$scope', '$filter', '$uibModal', '$rootScope',
-        'MisconceptionObjectFactory', 'EVENT_SKILL_REINITIALIZED',
+        'MisconceptionObjectFactory',
         'MAX_CHARS_IN_MISCONCEPTION_NAME',
-        function(
-            $scope, $filter, $uibModal, $rootScope,
-            MisconceptionObjectFactory, EVENT_SKILL_REINITIALIZED,
+        function( $scope, $filter, $uibModal, $rootScope,
```

Comment:
When stuff in parens spans a newline, break after the opening paren, like it was before.

Reply:
Got it, will change it.

Reply:
Not changed. Please check "Files Changed" before you request a review, and make sure you have responded to all open comments before doing so, otherwise things like this will delay the review process.

Reply:
Done!